src/CloudNimble.DotNetDocs.Sdk/Sdk/Sdk.props

```diff
@@ -73,23 +69,7 @@
         <MSBuildProjectExtensionsPath>$(BaseIntermediateOutputPath)</MSBuildProjectExtensionsPath>
```

Comment:
WOAH this is not cool - MSBuildProjectExtensionsPath is a private, very special property. I can only assume the engine didn't let you set this because the `dotnet` CLI sets it via env var, so it becomes un-overridable.